### ISSUE TITLE
Tchart can leverage tasklib

### ISSUE THREAD

**Issue by tbabej** [grovesteyn/tchart#6]
Tchart should use tasklib ( https://github.com/robgolding63/tasklib ) to simplify the code, since it provides pythonic access to TaskWarrior.

**Comment by grovesteyn** [comment]
Yes, thanks.  I am aware of tasklib and hope to use it when I get time 
to update tchart.

Regards

Grové
